plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/ConclaveExtension.kt

```diff
@@ -30,7 +30,7 @@ open class ConclaveExtension @Inject constructor(objects: ObjectFactory) {
     @get:Input
     val deadlockTimeout: Property<Int> = objects.property(Int::class.java).convention(10)
     @get:Input
-    val buildInDocker: Property<Boolean> = objects.property(Boolean::class.java).convention(true)
+    val useInternalDockerRepo: Property<Boolean> = objects.property(Boolean::class.java).convention(false)
```

Comment:
As discussed offline, this needs to be a separate flag and which is only needed by the `LinuxExec` task. Keep `buildInDocker` as is.